lib/scooter/httpdispatchers/peboltserver_dispatcher.rb

```diff
@@ -0,0 +1,18 @@
+module Scooter
+  module HttpDispatchers
+    class PEBoltServerDispatcher < HttpDispatcher
+
+      def initialize(host)
+        super(host)
+        @connection.url_prefix.port = 8144
+      end
+
+      # @return [Faraday::Response] response object from Faraday http client
+      def ssh_run_task(task)
+        @connection.post("ssh/run_task") do |req|
```

Comment:
Some of the other classes have a `set_default_path` method that saves the correct path up to the endpoint here; does it make sense to include that, or will you just set that in the tests?

Reply:
I don't think we have much of a path is why I didn't add it. This is just hitting "hostname/ssh/run_task" on port 8144.

Reply:
Ah, that does make sense. I think the `pe-bolt-server` would be a good candidate for that since our requests are so simple, and there won't be too many endpoints.  I'll give it a go in bolt and if I don't have it running by EOD this seems like the path of least resistance. But I like the idea of not having to make PRs to scooter for each endpoint :)